core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/pieChart/PieChartOpDesc.scala

```diff
@@ -28,12 +28,15 @@ import scala.collection.JavaConverters.asScalaBuffer
   */
 class PieChartOpDesc extends VisualizationOperator {
   @JsonProperty(value = "name column", required = true)
+  @JsonPropertyDescription("column of name (for chart label)")
   @AutofillAttributeName var nameColumn: String = _
 
   @JsonProperty(value = "data column")
+  @JsonPropertyDescription("column of data")
   @AutofillAttributeName var dataColumn: String = _
 
   @JsonProperty(value = "prune ratio", required = true)
+  @JsonPropertyDescription("names below this ratio will be grouped into one \"Other\" category")
```

Comment:
Quick comment, maybe give a default value in the FE so the user doesn't need to specify any

Reply:
Thanks! I add the default value for prune ratio (1) and chart type (pie) for pie chart as well as chart type (line) for line chart
<img width="336" alt="Screen Shot 2022-02-08 at 4 41 51 PM" src="https://user-images.githubusercontent.com/56903646/153100056-cc88128a-75ee-4a73-9d4b-3a93b8032b24.png">
<img width="338" alt="Screen Shot 2022-02-08 at 4 42 04 PM" src="https://user-images.githubusercontent.com/56903646/153100081-49913fe8-625a-4e74-b92c-ac1697627bc0.png">